src/correlation.rs

```diff
@@ -198,6 +198,24 @@ impl Correlations {
 
         Ok(())
     }
+
+    pub async fn delete_for_user(&self, user_id: &str) -> Result<(), CorrelationError> {
```

Comment:
```suggestion
    pub async fn delete_for_user(&self, user_id: &str) {
```

since we don't have any opportunities to error inside this function